spi/topology/src/main/java/quarks/topology/spi/graph/TWindowTimeImpl.java

```diff
@@ -1,14 +1,22 @@
-package quarks.topology.spi.graph;
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
 
-import static quarks.window.Policies.alwaysInsert;
```

Comment:
Any reason these were moved?

Eclipse puts the static imports first, but now they are after the regular imports.